src/m365/teams/commands/message/message-send.ts

```diff
@@ -0,0 +1,90 @@
+import { Logger } from '../../../../cli/Logger';
+import GlobalOptions from '../../../../GlobalOptions';
+import request, { CliRequestOptions } from '../../../../request';
+import { validation } from '../../../../utils/validation';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  teamId: string;
+  channelId: string;
+  message: string;
+}
+
+
+class TeamsMessageSendCommand extends GraphCommand {
+  public get name(): string {
+    return commands.MESSAGE_SEND;
+  }
+
+  public get description(): string {
+    return 'Sends a message to a channel in a Microsoft Teams team';
+  }
+
+  constructor() {
+    super();
+
+    this.#initOptions();
+    this.#initValidators();
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-i, --teamId <teamId>'
+      },
+      {
+        option: '-c, --channelId <channelId>'
+      },
+      {
+        option: '-m, --message <message>'
+      }
+    );
+  }
+
+  #initValidators(): void {
+    this.validators.push(
+      async (args: CommandArgs) => {
+        if (!validation.isValidGuid(args.options.teamId)) {
+          return `${args.options.teamId} is not a valid GUID`;
+        }
+
+        if (!validation.isValidTeamsChannelId(args.options.channelId as string)) {
+          return `${args.options.channelId} is not a valid Teams ChannelId`;
+        }
+
+        return true;
+      }
+    );
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    try {
+      const requestOptions: CliRequestOptions = {
+        url: `${this.resource}/v1.0/teams/${args.options.teamId}/channels/${args.options.channelId}/messages`,
+        headers: {
+          accept: 'application/json;odata.metadata=none',
+          'content-type': 'application/json;odata=nometadata'
+        },
+        responseType: 'json',
+        data: {
+          body: {
+            contentType: 'html',
+            content: args.options.message
+          }
+        }
+      };
+
+      await request.post(requestOptions);
```

Comment:
We're getting a response from Graph here, it might be useful to log this result.